modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling assisted installer service on bare metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units (DUs), {rh-rhacm} supports single node OpenShift deployments, which run on a single bare metal host. The single node cluster acts as both a control plane and a worker node. {rh-rhacm} supports single node OpenShift deployments for {product-title} on top of a single BMO DU.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+* Install {product-title} {product-verstion} on a hub cluster.
+* Install {rh-rhacm} and create the MultiClusterHub resource.
+* Create PersistentVolume custom resources (CR) for database and file system storage.
+* Install openshift-install CLI for the desired version of {product-title}.
+
+.Next Steps
+
+. Modify `HiveConfig` to enable the feature gate for Assisted Installer:
++
+[source,terminal]
+----
+ oc patch hiveconfig hive --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"}}}'
+----
+
+. Modify the `provisioning` resource to allow the bare metal operator to watch all namespaces:
++
+[source,terminal]
+----
+ oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+----
+
+. Create an `AgentServiceConfig` custom resource:
```

Comment:
You're not actually creating a resource with this command. Does the user need to store the CR in the GitOps repo (that I know exists because I've done a couple of reviews for ZTP content this sprint) or does the user need to do something like create the "custom resource that contains the following YAML:?

Reply:
Updated as suggested.